app/client/components/contactUs/contactUs.tsx

```diff
@@ -105,7 +106,13 @@ export const ContactUs = (props: ContactUsProps) => {
       email: formData.email,
       subject: formData.subjectLine,
       message: formData.details,
-      captchaToken: formData.captchaToken
+      captchaToken: formData.captchaToken,
+      ...(formData.fileAttachment && {
+        attachment: {
+          name: formData.fileAttachment.name,
+          contents: await base64FromFile(formData.fileAttachment)
```

Comment:
Can we move this logic into the `ContactUsForm` component to improve the separation of concerns? That way all the `ContactUs` component needs to concern itself with is to submit whatever it receives from `ContactUsForm`.